arch/i386/mcount-support.c

```diff
@@ -0,0 +1,239 @@
+#include <assert.h>
```

Comment:
I think this is a good place to add your authorship and copyright notation for this work. :)

Reply:
it is my honor.